test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py

```diff
@@ -2,16 +2,50 @@
 
 import pytest
 
-from test.test_utils import CONTAINER_TESTS_PREFIX
+from test.test_utils import CONTAINER_TESTS_PREFIX, PT_GPU_PY3_BENCHMARK_IMAGENET_AMI, DEFAULT_REGION
 from test.test_utils.ec2 import execute_ec2_training_performance_test
 
-
-PT_PERFORMANCE_TRAINING_GPU_SYNTHETIC_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_training_performance_gpu_synthetic")
+PT_PERFORMANCE_TRAINING_GPU_SYNTHETIC_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "benchmark",
+                                                         "run_pytorch_training_performance_gpu_synthetic")
+PT_PERFORMANCE_TRAINING_GPU_IMAGENET_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "benchmark",
+                                                        "run_pytorch_training_performance_gpu_imagenet")
 
 PT_EC2_GPU_SYNTHETIC_INSTANCE_TYPE = "p3.16xlarge"
+PT_EC2_GPU_IMAGENET_INSTANCE_TYPE = "p3dn.24xlarge"
 
 
 @pytest.mark.model("resnet50")
 @pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_SYNTHETIC_INSTANCE_TYPE], indirect=True)
 def test_performance_pytorch_gpu_synthetic(pytorch_training, ec2_connection, gpu_only, py3_only):
     execute_ec2_training_performance_test(ec2_connection, pytorch_training, PT_PERFORMANCE_TRAINING_GPU_SYNTHETIC_CMD)
+
+
+@pytest.mark.model("resnet50")
+@pytest.mark.parametrize("ec2_instance_ami", [PT_GPU_PY3_BENCHMARK_IMAGENET_AMI], indirect=True)
```

Comment:
why is a different AMI needed from regular us-east-1 AMI?

Reply:
The benchmark test toolkit and data set is pre-configured in this AMI. I am following the practice of manual steps for running this benchmark.